database/migrations/2019_03_08_204743_adds_callpower_campaign_id_unique_index_in_actions_table.php

```diff
@@ -26,7 +26,7 @@ public function up()
     public function down()
     {
         Schema::table('actions', function (Blueprint $table) {
-            $table->dropUnique('callpower_campaign_id');
+            $table->dropUnique(['callpower_campaign_id']);
```

Comment:
@chloealee I did the same for this migration for `callpower_campaign_id` since it was giving a similar error when rolling back! All is good now for this one too!

Reply:
ah thank you, @weerd !